src/main/java/com/etsy/statsd/profiler/profilers/JVMCPUProfiler.java

```diff
@@ -0,0 +1,114 @@
+package com.etsy.statsd.profiler.profilers;
+
+import com.etsy.statsd.profiler.Arguments;
+import com.etsy.statsd.profiler.Profiler;
+import com.etsy.statsd.profiler.reporter.Reporter;
+
+import java.lang.management.ManagementFactory;
+import java.util.concurrent.TimeUnit;
+import javax.management.Attribute;
+import javax.management.AttributeList;
+import javax.management.InstanceNotFoundException;
+import javax.management.MBeanServer;
+import javax.management.MalformedObjectNameException;
+import javax.management.ObjectName;
+import javax.management.ReflectionException;
+
+/**
+ * This profiler retrieves CPU values for the JVM and System from the "OperatingSystem" JMX Bean.
+ * <p>
+ * This profiler relies on a JMX bean that might not be available in all JVM implementations.
+ * We know for sure it's available in Sun/Oracle's JRE 7+, but there are no guarantees it
+ * will remain there for the foreseeable future.
```

Comment:
Is it feasible to detect the presence of the bean and fail fast if it isn't present?